src/use-utility-connect.js

```diff
@@ -3,10 +3,12 @@ import useScript from 'react-script-hook';
 
 const src = 'https://utility-connect-main.prod.arcadia.com/dist/v2.js';
 
-const scriptLoadError =
-  'Error fetching script - please check your internet connection.';
-const initializeError =
-  'Error loading Arcadia utility connect service - we are working to fix the issue.';
+const scriptLoadError = new Error(
+  'Error fetching script - please check your internet connection.'
+);
+const initializeError = new Error(
+  'Error loading Arcadia utility connect service - we are working to fix the issue.'
```

Comment:
Sorry I'm a bit late here - would be good to capitalize `Utility Connect` anywhere that it would get surfaced to partners.